Index.html

```diff
@@ -8,60 +8,63 @@
     <title>Renome restaurant</title>
   </head>
 
-  <main>
-    <body>
-      <div class="navigation">
-        <p class="navigation-left">Renome</p>
+  <body>
+    <header>
+      <nav class="navigation">
+        <p class="navigation-logo">Renome</p>
         <div class="navigation-right">
           <img
-            class="navigation-right-cart"
+            class="navigation-right_cart"
```

Comment:
Is there a navigation-left? Also should use two underscores for separation "navigation-right__cart"